packages/core/database/lib/index.js

```diff
@@ -59,13 +59,16 @@ class Database {
       if (notNestedTransaction) {
         await trx.commit();
       }
+      transactionCtx.clear();
```

Comment:
I would suggest to move this to the if statement, otherwise it will break the nested transactions and the context will be cleared when it shouldn't